cpp/ql/lib/change-notes/2022-03-14-taint-interface-cleanup.md

```diff
@@ -0,0 +1,4 @@
+---
+category: breaking
+---
+* The flow state variants of `isBarrier` and `isAdditionalFlowStep` are no longer exposed in the taint tracking library. The `inSanitizer` and `isAdditionalTaintStep` predicates should be used instead.
```

Comment:
Typo in all the change notes:

```suggestion
* The flow state variants of `isBarrier` and `isAdditionalFlowStep` are no longer exposed in the taint tracking library. The `isSanitizer` and `isAdditionalTaintStep` predicates should be used instead.
```

Reply:
I'm, not awake yet apparently. Thanks for spotting that.

Reply:
Fixed.